eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -78,12 +83,27 @@ function GetAdjustedReadmeContent($ReadmeContent, $PackageInfo, $PackageMetadata
     $ReadmeContent = $ReadmeContent -replace $releaseReplaceRegex, $replacementPattern
   }
   
+  $author = "ramya-rao-a"
+  $msauthor = "ramyar"
+  # We are currently lack of the ability to get ms alias from github identity. Use github identity as placeholder for now.
+  if ($CodeOwners) {
+    $author = $CodeOwners.Split(",")[0]
+    $msauthor = $author 
+  }
+  elseif (Test-Path '$PSScriptRoot/default-code-owner.json') {
```

Comment:
Where is this file coming from?

Reply:
I am planning to put it under eng/default-code-owner.json